experiments/5.0-je-v1-streamlistener.py

```diff
@@ -0,0 +1,73 @@
+from tweepy.streaming import StreamListener
+from tweepy import OAuthHandler, Stream
+from twitter_keys_tokens import keys_tokens
+import time
+import csv
+import sys
+
+
+class TweetListener(StreamListener):
+    def __init__(self, api=None):
+        self.api = api
+        self.filename = "data_" + time.strftime("%Y%m%d") + ".csv"
+        csvfile = open(self.filename, "w")
+        csvwriter = csv.writer(csvfile)
+        csvwriter.writerow(
+            [
+                "created_at", "id", "user_id", "username", "name", "tweet",
+                "replies_count", "retweets_count", "likes_count"
+            ]
+        )
+
+    def on_status(self, status):
+        csvfile = open(self.filename, "a")
+        csvwriter = csv.writer(csvfile)
+        if "RT @" not in status.text:
+            try:
+                csvwriter.writerow(
+                    [
+                        status.created_at, status.id_str, status.user.id_str,
+                        status.user.screen_name, status.user.name, status.text,
+                        status.reply_count, status.retweet_count,
+                        status.favorite_count
+                    ]
+                )
+            except Exception as e:
+                print(e)
+                pass
+        csvfile.close()
+        return
+
+    def on_error(self, status_code):
+        print("Encountered error with status code:", status_code)
```

Comment:
From now on, use only python format strings for printing. This one would be written instead as:

```
print(f"Encountered error with status code: {status_code}")
```

Also, minor note: Python style prefers the single quote over the double quote by default. Most important thing here is to be consistent, but just be aware. Some people are more picky on this stuff than I am.